contracts/.solhint.json

```diff
@@ -3,11 +3,10 @@
   "plugins": ["prettier"],
   "rules": {
```

Comment:
Another issue on withdrawal on Optimism:

The fee estimate is shown in scientific notation (maybe that's ok?) but when I click the button to edit it, I get an error:

<img width="1524" alt="Screen Shot 2022-03-17 at 11 18 01 PM" src="https://user-images.githubusercontent.com/1479802/158931166-e0399fe2-2330-4779-aea4-2455d1bdb710.png">

Reply:
From some quick google searches, it seems we'll have to write our own handling of very small numbers to avoid them being handled with scientific notation, which seems like a pain. For now I just pushed 8f2b392 which adds a failing test but perhaps we pull this into a separate issue?

Reply:
Part of the issue here is that the 2.1e-8 fee that is being shown is just the L2 execution fee, but the L1 brings the fee total to roughly `0.00018`, which probably would not get shown as scientific notation. So we should update the confirm withdrawal modal to use the new `getEthSweepGasInfo` method from `Umbra.ts`, which returns a `txCost` value that should be shown

Reply:
Got it. Yeah, I'm not even sure we'd want to show a number like 2.1e-8 as `0.00000002.1`, it would end up looking pretty goofy anyway. But if the real fee is actually higher, we should show be showing that anyway, which sounds like it fixes the scientific notation problem.

Reply:
The fee should now be properly displayed as of 4788b35, but I still need to fix the edit button

Reply:
Looks good 👍